index.d.ts

```diff
@@ -763,12 +763,20 @@ declare namespace Moleculer {
 		logger: LoggerInstance;
 		actions: ServiceActions;
 		Promise: PromiseConstructorLike;
-		//currentContext: Context | null;
 
 		_init(): void;
```

Comment:
With eliminating the "internal" methods, I believe these should be removed as well.

Reply:
I kept it because it was eliminated early, and can cause breaking if I remove it and somebody uses them. So I removed that is added in this PR only.